contrib/kakoune.rb

```diff
@@ -1,17 +1,20 @@
-require "formula"
-
 class Kakoune < Formula
   homepage "https://github.com/mawww/kakoune"
   head "https://github.com/mawww/kakoune.git"
 
-  depends_on 'docbook-xsl' => :build
-  depends_on 'ncurses' => [:build, :recommended]
-  depends_on 'asciidoc' => [:build, 'with-docbook-xsl']
```

Comment:
Why was this option removed?

Reply:
Because it [no longer](https://github.com/Homebrew/homebrew-core/commit/90aae3e04c9af9d834c16c6820aff930fed5c3f4) exists.